src/Umbraco.Core/Configuration/HealthChecks/DisabledHealthCheckElement.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Configuration;
+using System.Linq;
+using System.Text;
+
+namespace Umbraco.Core.Configuration.HealthChecks
+{
+    public class DisabledHealthCheckElement : ConfigurationElement
+    {
+        private const string ID_KEY = "id";
+        private const string DISABLED_ON_KEY = "disabledOn";
```

Comment:
These are not correct naming conventions for constants - though they are private so it's not a big deal